pkg/controllers/management/aks/aks_cluster_handler.go

```diff
@@ -260,6 +248,22 @@ func (e *aksOperatorController) onClusterChange(key string, cluster *mgmtv3.Clus
 	}
 }
 
+// check if any resources in aks.cattle.io/v1
```

Comment:
Can you fix this comment?
```suggestion
// checkCRDReady checks whether necessary CRD, AKSConfig, has been created yet
```

Reply:
:+1: fixed